src/app/pages/settings/contentSettings.js

```diff
@@ -1,181 +1,217 @@
+			__(
+				'The trash will automatically empty every ',
+				'wp-plugin-bluehost'
+			) +
 			numTrashWeeks +
-			_n( ' week.', ' weeks.', parseInt(numTrashWeeks), 'wp-plugin-bluehost' )
+			_n(
+				' week.',
+				' weeks.',
+				parseInt( numTrashWeeks ),
+				'wp-plugin-bluehost'
+			)
 		);
 	};
 
-	const handleEmptyTrashDaysChange = (value) => {
-		bluehostSettingsApiFetch({ emptyTrashDays: value }, setError, (response) => {
-			setNumEmptyTrashDays(value);
-		});
+	const handleEmptyTrashDaysChange = ( value ) => {
+		bluehostSettingsApiFetch(
+			{ emptyTrashDays: value },
+			setError,
+			// eslint-disable-next-line no-unused-vars
+			( response ) => {
+				setNumEmptyTrashDays( value );
+			}
+		);
 	};
 
 	const notifySuccess = () => {
-		notify.push("empty-trash-notice", {
+		notify.push( 'empty-trash-notice', {
 			title: emptyTrashNoticeTitle(),
-			description: (
-				<span>
-					{emptyTrashNoticeText()}
-				</span>
-			),
-			variant: "success",
+			description: <span>{ emptyTrashNoticeText() }</span>,
+			variant: 'success',
 			autoDismiss: 5000,
-		});
+		} );
 	};
 
-	useUpdateEffect(() => {
-		setStore({
+	useUpdateEffect( () => {
+		setStore( {
 			...store,
 			emptyTrashDays,
-		});
+		} );
 		numTrashWeeks = Math.floor( emptyTrashDays / 7 );
 
 		notifySuccess();
-	}, [emptyTrashDays]);
+	}, [ emptyTrashDays ] );
 
 	return (
 		<SelectField
 			id="empty-trash-select"
-			label={__('Trash emptying frequency ', 'wp-plugin-bluehost')}
+			label={ __( 'Trash emptying frequency ', 'wp-plugin-bluehost' ) }
 			description={
-			__('The trash will automatically empty every ', 'wp-plugin-bluehost') +
-			numTrashWeeks +
-			 _n( ' week.', ' weeks.', parseInt(numTrashWeeks), 'wp-plugin-bluehost' )
+				__(
+					'The trash will automatically empty every ',
+					'wp-plugin-bluehost'
+				) +
+				numTrashWeeks +
+				_n(
+					' week.',
+					' weeks.',
+					parseInt( numTrashWeeks ),
+					'wp-plugin-bluehost'
+				)
 			}
-			value={emptyTrashDays}
-			selectedLabel={numTrashWeeks}
-			options={[
+			value={ emptyTrashDays }
+			selectedLabel={ numTrashWeeks }
+			options={ [
 				{ label: '1', value: '7' },
 				{ label: '2', value: '14' },
 				{ label: '3', value: '21' },
 				{ label: '4', value: '30' },
-			]}
-			onChange={handleEmptyTrashDaysChange}
+			] }
+			onChange={ handleEmptyTrashDaysChange }
 			className="nfd-select-field__spaced"
 		/>
 	);
-}
+};
 
 const ContentSettings = () => {
-	const [isError, setError] = useState(false);
+	const [ isError, setError ] = useState( false );
 
-	let notify = useNotification();
+	const notify = useNotification();
 	return (
 		<SectionSettings
-			title={__('Content Options', 'wp-plugin-bluehost')}
-			description={__('Controls for content revisions and how often to empty the trash.', 'wp-plugin-bluehost')}
+			title={ __( 'Content Options', 'wp-plugin-bluehost' ) }
+			description={ __(
+				'Controls for content revisions and how often to empty the trash.',
```

Comment:
We say "content revisions" here but "post revisions" elsewhere, let's make it all consistent. 

I prefer "content", but I think that implies that it's _all_ content, including things like pages or site tagline or menus or something. 🤔 Not sure what's best.